modules/otel-config-instrumentation.adoc

```diff
@@ -344,3 +344,46 @@ instrumentation.opentelemetry.io/container-names: "<container_1>,<container_2>"
 ====
 The Go auto-instrumentation does not support multi-container auto-instrumentation injection.
 ====
+
+[id="otel-multi-container-pods-multi-instrumentations_{context}"]
+=== Multi-container pods with multiple instrumentations
+
+Injecting instrumentation for an application language to one or more containers in a multi-container pod requires the following annotation:
+
+[source,yaml]
+----
+instrumentation.opentelemetry.io/<application_language>-container-names: "<container_1>,<container_2>" # <1>
+----
+<1> You can only inject instrumentation for one language per container. For the list of supported `<application_language>` values, see the following table.
```

Comment:
"only" should be as close as possible to the term it modifies.

```suggestion
<1> You can inject instrumentation for only one language per container. For the list of supported `<application_language>` values, see the following table.
```

Reply:
Thank you. Fixed.